includes/MediaWiki/ReconciliationException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki;
+
+use Exception;
+use Wikimedia\Message\MessageValue;
+
+/**
+ * Base class for all WikibaseReconcileEdit exceptions
+ *
+ * @license GPL-2.0-or-later
+ */
+class ReconciliationException extends Exception {
+
```

Comment:
The class should also declare `private $messageValue` with a suitable `@var` comment.

Reply:
True! Thanks.